frontend/src/test/scala/bloop/tasks/IntegrationTestSuite.scala

```diff
@@ -25,25 +25,9 @@ object IntegrationTestSuite extends DynTest {
   private def testProject(testDirectory: Path, logger: Logger): Unit = {
     val rootProjectName = "bloop-test-root"
     val classesDir = AbsolutePath(testDirectory)
-    val projects = {
-      val projects = ProjectHelpers.loadTestProject(testDirectory.getFileName.toString, logger)
-      val rootProject = Project(
-        name = rootProjectName,
-        baseDirectory = AbsolutePath(testDirectory),
-        dependencies = projects.keys.toArray,
-        scalaInstance = projects.head._2.scalaInstance,
-        classpath = Array.empty,
-        classesDir = classesDir,
-        scalacOptions = Array.empty,
-        javacOptions = Array.empty,
-        sourceDirectories = Array.empty,
-        previousResult = CompilationHelpers.emptyPreviousResult,
-        tmp = classesDir,
-        testFrameworks = Array.empty,
-        origin = None
-      )
-      projects + (rootProjectName -> rootProject)
-    }
+    val projects = ProjectHelpers.loadTestProject(testDirectory.getFileName.toString,
```

Comment:
Is it possible to avoid doing these changes? I change these things quite a bit, and looks unrelated to the PR?